### PR TITLE
Feature/speak tts audio file

### PR DESCRIPTION
**Pull request by JackLivio**
Implements text to speech for audio files and text ttsChunk types.

### CODE REVIEW

File: src/js/Controllers/TTSController.js [hunk 1]
```diff
@@ -18,6 +88,20 @@ class TTSController {
...
+                }
+
+                if(this.filePlaylist.length > 0) {
+                    if(this.filePlaylist[0].type === "FILE") {
+                        this.playAudio();
+                    } else if (this.filePlaylist[0].type === "TEXT"){
+                        this.speak();
+                    }
+                }
+
```
Comment (jacobkeeler) [inline src/js/Controllers/TTSController.js]: Should also be added to these (assuming they are implemented):
1. `Alert` 
2. `PerformAudioPassThru`
3. `AlertManeuver`

File: src/js/Controllers/TTSController.js [hunk 2]
```diff
@@ -2,10 +2,80 @@ import RpcFactory from './RpcFactory'
...
+        this.audioPlayer.onended = () => {
+            this.audioPlayer.src ="";
+            if(this.filePlaylist[0]) {
+                if(this.filePlaylist[0].type === "FILE") {
+                    this.playAudio();
+                } else if (this.filePlaylist[0].type === "TEXT"){
+                    this.speak();
+                }
+            }
+        }
```
Comment (jacobkeeler) [inline src/js/Controllers/TTSController.js]: Should have an error function for this as well, sometimes the audio file type is not supported.

File: src/js/Controllers/TTSController.js (hunk 1 above)
Comment (JackLivio) [reply src/js/Controllers/TTSController.js]: These are covered by TTS.SPEAK on the HMI side

File: src/js/Controllers/TTSController.js (hunk 2 above)
Comment (JackLivio) [reply src/js/Controllers/TTSController.js]: updated

**Pull request merged**